server/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = pep8,py{36,38}-inm{master,iso4-stable,latest}
+envlist = pep8,py36-inmiso4,py39-inm{master,iso{4,5}-stable,latest}
```

Comment:
Why also run `iso4` using `python39`?

Reply:
Good point, it's not OSS so I guess 3.6 suffices.